source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -1643,6 +1650,12 @@ void Filter::mergePerRouteConfig() {
     decoding_state_.setUntypedReceivingMetadataNamespaces(untyped_receiving_namespaces_);
     encoding_state_.setUntypedReceivingMetadataNamespaces(untyped_receiving_namespaces_);
   }
+
+  if (merged_config->failureModeAllow().has_value()) {
```

Comment:
optional: do you want to test merged_config with multiple routes as well? Probably it can be done in next PR

Reply:
Yeah, I've added a test for this. Thanks.